polymer.externs.js

```diff
@@ -666,10 +739,84 @@ Polymer.Templatizer = {
   /**
    * @param {?Element} template
    */
-  templatize: function(template) {}
+  templatize: function(template) {},
+
+  /**
+   * Returns the template "model" associated with a given element, which
+   * serves as the binding scope for the template instance the element is
+   * contained in. A template model is an instance of `Polymer.Base`, and
+   * should be used to manipulate data associated with this template instance.
+   *
+   * Example:
+   *
+   *   var model = modelForElement(el);
+   *   if (model.index < 10) {
+   *     model.set('item.checked', true);
+   *   }
+   *
+   * @param {!HTMLElement} el Element for which to return a template model.
+   * @return {Object<PolymerElement>} Model representing the binding scope for
```

Comment:
Looks like the intent was to say that the returned object implements or extends Polymer.Base. In our type system I think that most closely corresponds to PolymerElement.

Here's the source to double-check this: https://github.com/Polymer/polymer/blob/8868fdc9f231c615e554ec21833efe0d62de4718/src/lib/template/templatizer.html#L411